src/application/tabs.css

```diff
@@ -99,18 +99,18 @@
 }
 
 
-.p-TabBar.p-mod-top .p-TabBar-tab {
+.p-TabBar.jp-mod-top .p-TabBar-tab {
   border-bottom: none;
 }
 
 
-.p-TabBar.p-mod-bottom .p-TabBar-tab {
+.p-TabBar.jp-mod-bottom .p-TabBar-tab {
   border-top: none;
 }
 
 
-.p-TabBar.p-mod-top .p-TabBar-tab:first-child,
-.p-TabBar.p-mod-bottom .p-TabBar-tab:first-child {
+.p-TabBar.jp-mod-top .p-TabBar-tab:first-child,
+.p-TabBar.jp-mod-bottom .p-TabBar-tab:first-child {
```

Comment:
For now, I think I should leave them to apply to all tabs unless we find that we need to be more restrictive.